AdvancedDLSupport/NativeLibraryBase.cs

```diff
@@ -51,38 +52,43 @@ public abstract class NativeLibraryBase : IDisposable
         /// Gets the library and symbol loader for the current platform.
         /// </summary>
         [NotNull]
-        private static readonly IPlatformLoader PlatformLoader;
+        private static IPlatformLoader PlatformLoader => PlatformLoaderBase.PlatformLoader;
```

Comment:
This field can be removed. It is not used internally other than in a single assignment.